### PR TITLE
[New Package]: MatsuriDayo.NekoRay 3.25

### PR DESCRIPTION
**Pull request by Erdenian**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/128057)

### CODE REVIEW

**Comment by stephengillie** [comment]
Manual Validation ended with:
image

File: manifests/m/MatsuriDayo/NekoRay/3.25/MatsuriDayo.NekoRay.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: MatsuriDayo.NekoRay
+PackageVersion: "3.25"
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: nekoray\nekoray.exe
+  PortableCommandAlias: nekoray
```
Comment (stephengillie) [inline manifests/m/MatsuriDayo/NekoRay/3.25/MatsuriDayo.NekoRay.installer.yaml:10]:
```suggestion
  PortableCommandAlias: nekoray
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

Comment (Erdenian) [reply manifests/m/MatsuriDayo/NekoRay/3.25/MatsuriDayo.NekoRay.installer.yaml:10]: Thank you

**Review by stephengillie — APPROVED** [review]

**Pull request merged**